docs/topics/settings.txt

```diff
@@ -231,7 +231,7 @@ Normally, you will not need to override the defaults in this fashion. The
 Django defaults are sufficiently tame that you can safely use them. Be aware
 that if you do pass in a new default module, it entirely *replaces* the Django
 defaults, so you must specify a value for every possible setting that might be
-used in that code you are importing. Check in
+used in the code you are importing. Check-in
```

Comment:
I don't think the hyphen is great there. We're saying to check (one thing) in (another thing) this folder. We're not referring to a single thing, such as a _check-in_ you might do on a social media app (say). 


the `s/that/the/` seems right. 👍